modules/packages/Sort.chpl

```diff
@@ -263,14 +433,15 @@ proc QuickSort(Data: [?Dom] ?eltType, minlen=16, doublecheck=false, param revers
   Data(loptr) = pivotVal;
 
   //  cobegin {
```

Comment:
Not sure, this is how I found it... I will leave this change to another PR as well.